### PR TITLE
Add class referencing base layer to allow styling

### PR DESCRIPTION
**Pull request by danielfdsilva**

### CODE REVIEW

**Comment by danielfdsilva** [comment]
@smit1678 I added a class that identifies the current active base layer and allows you to define specific styles.

File: app/assets/styles/_map.scss [hunk 1]
```diff
@@ -47,6 +47,18 @@
...
+  // Style cell shading according to layer
+  .base-layer--oam-base & {
+    fill: $primary-color;
+  }
+  .base-layer--osm & {
+    fill: $primary-color;
+  }
+  .base-layer--satellite & {
+    fill: #FFF;
+  }
```
Comment (danielfdsilva) [inline app/assets/styles/_map.scss]: Define here the different fills. For now I just used the white for the satellite layer

**Comment by smit1678** [comment]
@danielfdsilva Can you give d09a055 a quick review? Better way to handle this?

**Comment by danielfdsilva** [comment]
@smit1678 that ought to do it.

**Comment by smit1678** [comment]
👍  thanks

**Pull request merged**